### ISSUE TITLE
training model did not improve accuracy!

### ISSUE THREAD

**Issue by spate141** [cmusphinx/g2p-seq2seq#16]
So, I was training a new model based on CMUSphinx dictionary, with `--max_steps 10000 --size 512 --num_layers 3 --learning_rate 0.5` variables, after I finished the training on model, I got this output with trained model.

> a
M HH HH HH HH UH UH UH UH UH
> b
M M UH UH UH UH UH UH UH UH
> c
M M M UH UH UH UH UH UH UH
> d
M M M M UH UH UH UH UH UH
> hello
M HH HH HH HH UW UW UW UW M M M M M M
> aa
HH HH HH HH HH HH HH HH UH UH

Is there anything wrong with my approach ?

this was my last output in training model.
`global step 10000 learning rate 0.4000 step-time 3.48 perplexity 1.15`
`eval: bucket 0 perplexity 1.40`
`eval: bucket 1 perplexity 1.25`
`eval: bucket 2 perplexity 1.34`

**Comment by nurtas-m** [comment]
Hello!
I think that when you start interactive mode, trained model parameters not match to default model's parameters.
Try to run interactive mode with parameters that you previously use to train model:
python g2p.py --interactive --max_steps 10000 --size 512 --num_layers 3 --learning_rate 0.5 --model path_to_model_directory

**Comment by spate141** [comment]
I tried it, but still no luck.

**Comment by nurtas-m** [comment]
Can you, please, provide command that you ran on training process and on interactive process.

**Comment by spate141** [comment]
Training: `python g2p.py --train cmudict.dict --model train --max_steps 10000 --size 512 --num_layers 3 --learning_rate 0.5`

Interactive process: `python g2p.py --interactive --model train --max_steps 10000 --size 512 --num_layers 3 --learning_rate 0.5`

**Comment by nurtas-m** [comment]
1. When you start interactive mode is there message like "Created model with fresh parameters"?
Seems like program not find trained model and simply load fresh model. 
2. In model_dir directory there should be next files:
checkpoint,
translate.ckpt-9200,
...
translate.ckpt-10000,
translate.ckpt-9200.meta,
...
translate.ckpt-10000.meta,
vocab.grapheme,
vocab.phoneme
Can you check that all this files are in model_dir directory (remove or exclude all other additional files or subdirectories from the model_dir)?
 
3. Checkpoint file should be with following content:
model_checkpoint_path: "translate.ckpt-10000"
all_model_checkpoint_paths: "translate.ckpt-9200"
all_model_checkpoint_paths: "translate.ckpt-9400"
all_model_checkpoint_paths: "translate.ckpt-9600"
all_model_checkpoint_paths: "translate.ckpt-9800"
all_model_checkpoint_paths: "translate.ckpt-10000"
Can you check it?

**Comment by spate141** [comment]
1. Yes, there was a message saying `Created model with fresh parameters.`
2. No, I did removed all files and was only using last file `translate.ckpt-10000` (my mistake)
3. No, as I said early, I had just phonemes, graphemes and last checkpoint file in test_model dir, It was not working.

solution: as mentioned be @nurtas-m : model dir has now all files created during training phase, everything seems working well. Thank you. 

@nurtas-m 👍
